examples/.gitignore

```diff
@@ -1,2 +1,3 @@
+/*/src/.observablehq/deploy.json
```

Comment:
I don’t want to commit these to git because when users clone these examples, they aren’t going to want them already configured to deploy to our workspace. It’d be nice if we could just commit them for our own private use but I couldn’t think of a way to do that.

Reply:
Yes, I’d like to figure that out in the future.